dev-libs/libksba/libksba-1.3.5.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="Library for working with X.509 certificates, CMS data and related"
+DESCRIPTION="Libksba is a library to make the task of working with X.509, CMS \
+data and related objects easier. It provides a highlevel interface to \
+implemented protocols and presents the data in a consistent way."
+HOMEPAGE="https://www.gnupg.org/related_software/libksba/"
+COPYRIGHT="2001-2015 g10 Code GmbH"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://www.gnupg.org/ftp/gcrypt/libksba/libksba-$portVersion.tar.bz2"
+CHECKSUM_SHA256="41444fd7a6ff73a79ad9728f985e71c9ba8cd3e5e53358e70d5f066d35c1a340"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
```

Comment:
@xdizzaster, this means you just need to add
```
SECONDARY_ARCHITECTURES="x86_gcc2 x86"
```
and insert the famous $secondaryArchSuffix at the usual places.

Great job indeed!